packages/flame/lib/src/components/mixins/hoverable.dart

```diff
@@ -35,12 +35,13 @@ mixin HasHoverableComponents on BaseGame {
       return true; // always continue
     }
 
-    for (final c in components.reversed()) {
-      if (c is BaseComponent) {
-        c.propagateToChildren<Hoverable>(_mouseMoveHandler);
+    for (final c in children.reversed()) {
```

Comment:
this is another piece of code that seems to be repeated a lot. I think there must be a better abstraction than `propagateToChildren` that requires each place to have less code...

Reply:
Opened #949 for that, since it isn't really related to this PR.